src/schema/objects/columns.yaml

```
@@ -403,6 +421,13 @@ reference_frame:
     - type: string
       enum:
         - n/a
+region:
+  name: region
+  display_name: Region
+  description: |
+     "XY", where X can be L:left, R:right, B:bilateral, and Y can be F:frontal, T:temporal, P:parietal, O:occipital
+  type: string
+  pattern: ^[L|l|R|r|B|b][F|f|T|t|P|p|O|o]$
```

`|` is used in parentheses. For character groups, just cluster them:

```suggestion
  pattern: ^[LlRrBb][FfTtPpOo]$
```

Do you really want to allow mixed case? IMO when you're introducing a thing is a time to be strict as it's not harder for someone to use LT than lt.

---

This was inherited from the original BEP -- I agree with you about the mixed case, can I edit your suggestion?

---

Go for it

---

Not sure what's going on. Doing it on my computer. BTW - these are columns. Did you mean lower or upper case?

---

These are values, so I guess you meant uppercase. For some reason, my GitHub view is outdated. Pulling the commit seems to have correctly applied the change.

---

I think caps is clearer that they're initialisms, and easier to ready quickly. (And if you're mentioning columns because the convention is for snake_case columns, that's just the headers. There's no convention on values that I've noticed.)